webpack.config.js

```diff
@@ -32,6 +32,19 @@ module.exports = {
     filename: "[name].js",
     path: path.resolve(__dirname, "Client/Assets")
   },
+  module: {
+    rules: [
+      {
+        test: /\.js$/,
+        use: {
+          loader: "babel-loader",
+          options: {
+            presets: ["es2015"]
+          }
+        }
+      }
+    ]
+  },
```

Comment:
Put `babel-loader` in the a `loaders` section.

```js
    module: {
        loaders: [{
            test: /\.js$/,
            loader: 'babel-loader',
            exclude: [
                /node_modules\/babel-/m,
                /node_modules\/core-js\//m,
                /node_modules\/regenerator-runtime\//m
            ]
        }]
    }
```

Then put the `babel-preset-env` in `.babelrc`.

```js
{
  "presets": [
    ["env", {
      "targets": {
        "iOS": "10.0"
      }
    }]
  ],
  "plugins": ["transform-runtime"]
}
```

Though, looking at [babel-preset-env][1] docs, I'm not sure `"env"` is the right string here.

 [1]: https://github.com/babel/babel/tree/master/packages/babel-preset-env

Reply:
I effectively did the same thing you suggested here, except rolled the .babelrc stuff into the webpack.config. The syntax seems to have evolved here over time (like most JS tooling). What I used comes from the latest docs as far as I can tell.